foyer/tests/implemented_opls_tests.txt

```diff
@@ -35,22 +36,29 @@ diethylene-glycol
 diethylene-glycol-dimethyl-ether
 dimethylether
 dimethylformamide
+ethane
 ethanol
 ethylbenzene
 fluorobenzene
 formic-acid
 isobutane
+methane
 methanol
 n-butylamine
+N-methylacetamide
+neopentane
 nitrobenzene
 nitroethane
 nitromethane
+nmp
```

Comment:
Please use the full name in case something else may be interpreted as `nmp` in the future

Reply:
On it! I'm used to it as NMP but better safe than sorry. I'm changing the appropriate files to N-methylpyrrolidone.

I didn't have a reason to change it from GRO/TOP to MOL2, per say. I just made those mol2 files from scratch and double-checked the atomtypes. I also wasn't aware of the issue with ParmEd conversions but I did not convert those GRO/TOP  files directly to MOL2. Do you think it would be better to leave them in the original GRO/TOP?